biotope/2018-06-05_biotopeVision.md

```diff
@@ -0,0 +1,22 @@
+# Biotope is a framework and architecture which enables you to develop technology and cms independent frontend component libraries
+
+
+## It helps developers to:
+- produce clean and state of the art frontend code
+- develope ui-components on a unified standard and with latest web standards
+- keep focus on developing components, the architecture and dev environment is provided by Biotope.
+- make it easy to work with a team of many developers on the same component library
+- ensure high quality standards for the whole library even when developed by multiple developers
+- build real projects and design system styleguides with a single code source
+- create your own ui library to chose from in your projects
+- use Biotope ui-components in single page applications as well as in traditional server side renderd cms
+
+
+## It helps companies to:
+- Create a UI component library which can be maintained in a central place
+- BUT all ui-components can be used independently in different projects
+- Create UI consistency over all projects
+- Make it possbile to let multiple partners contribute to the ui library without compatiblity & quality issues
+-	Bring their Design System to live. Real usable code, no dummy html components in a seperate styleguide tool.
+-	Doesnt matter how big the component library gets. Projects only use components they need. So performance stays high.
+- Maintain and extend the ui library over years without the need for a relaunch
```

Comment:
...without the need for rework (i think relaunch is too specific)

Reply:
is restart ok? because in fact rework has maybe to be done (be compatible with new biotope versions, latest web standards, ...) but there is no need to start again from scratch after a few years because your code is a mess or got too big to be maintained

Reply:
Why not write "keeping it up-to-date and usable" or "to avoid getting out-of-date and having to start from scratch again"?